src/components/CharacterOrderDialog.vue

```diff
@@ -145,6 +148,7 @@
                           voiceSampleIndex
                         );
                       "
+                      :aria-label="`ボイスサンプル${voiceSampleIndex + 1}`"
```

Comment:
webページではサンプルボイスと呼んでいるのでそうさせていただきます！
```suggestion
                      :aria-label="`サンプルボイス${voiceSampleIndex + 1}`"
```